site/app/libraries/database/DatabaseQueries.php

```diff
@@ -600,7 +600,14 @@ public function loadThreadBlock($categories_ids, $thread_status, $unread_threads
         }
         $query_offset = ($blockNumber - 1) * $blockSize;
         $this->buildLoadThreadQuery($categories_ids, $thread_status, $unread_threads, $show_deleted, $show_merged_thread, $current_user, $query_select, $query_join, $query_where, $query_order, $query_parameters, true, true);
-        $query = "SELECT {$query_select} FROM threads t {$query_join} WHERE {$query_where} ORDER BY {$query_order} LIMIT ? OFFSET ?";
+        if ($sortOption == "time") {
```

Comment:
It's generally better to use strict equality in PHP
```suggestion
        if ($sortOption === "time") {
```